docs/example-narrative-in-use-forecasts.md

```diff
@@ -16,65 +20,24 @@ curl -H "Accept: application/vnd.trolie.forecast-limit-set-slim.v1+json" \
 
 This will return the current version of the in-use ratings for the next 240 hours into output.json.  See the following for an example:
 
+### Reuse JSON method 1:
+
+ {% capture example %}
+ {{- site.data.paths["limits_forecast-snapshot"].get.responses["200"].content["application/vnd.trolie.forecast-limit-set-slim.v1+json"].example["$ref"] 
+    | split: "/" | last
+ -}}
+ {% endcapture %}
+
+{% assign example_json = '_data/examples/' | append: example %}
+
+```json
+{% include_relative {{example_json}} %}
+```
```

Comment:
This has the virtue of using exactly the `_data/examples/...json` used in the api spec, but the downsides are pretty clear: It assumes the `_data/examples` convention and that capture is pretty ugly.